docs/installation/ubuntulinux.md

```diff
@@ -146,7 +146,7 @@ To upgrade your kernel and install the additional packages, do the following:
 
 5. After your system reboots, go ahead and install Docker.
 
-
+>**Note**:  In order for Docker to start on Ubuntu 14.04 or lower, apparmor needs to be installed. It can be downloaded using the following command: `apt-get install apparmor`
```

Comment:
Does apparmour come pre-installed on other versions of ubuntu? It might be a good idea to look into it. 

If it is already pre-installed then it might a good idea to leave it as a note.  What do you guys think?